app/views/render_page.scala.html

```diff
@@ -24,17 +23,7 @@
 
 @layout(messages("dummy.title")) {
```

Comment:
The message key "dummy.title" is hardcoded into the render. If we are going to use this template going forward I think the key should be passed in by the controller.

Reply:
Mac mentioned there is some other way of doing this, so I will remove the line and push to the branch